client/src/components/SubmitForm.js

```diff
@@ -1,9 +1,13 @@
-export default function SubmitForm({ handleOnSubmit }) {
+export default function SubmitForm({ handleOnSubmit, className }) {
   return (
-    <form onSubmit={handleOnSubmit}>
-      <label htmlFor="searchInput">PLZ oder Stadteil</label>
-      <input name="searchInput" id="searchInput" />
-      <button type="submit">SEND</button>
+    <form className={`submitform ${className}`} onSubmit={handleOnSubmit}>
+      <input
+        required
+        placeholder="Ort oder Postleitzahl"
```

Comment:
> Consider using a Label as well, otherwise people won't be able to remember what field they are working in as soon as they wrote anything in it.

I mean there is only 1 field. is it in that case still a major thing to not use one?